### ISSUE TITLE
The player progress bars become unstable for large media files

### ISSUE THREAD

**Issue by michc** [axiopisty/plarpebu#1]
The player progress bars become unstable for large media files due to the limitations of using an int parameter for the number of samples in the progress method of several plugins. This can be fixed by changing the first parameter to a long but requires that the javazoom.jlgui.basicplayer.BasicPlayerListener library component be updated to match.

**Comment by axiopisty** [comment]
Yes. You are correct. Are you running this with the ant build from the master branch?

**Comment by michc** [comment]
No. I was using the build-with-gradle branch. I have done the mod.s to the
afeected classes and made a modified basicplayer3.0.jar. It works.
